### ISSUE TITLE
report‘s monitor.html lack some node infomation

### ISSUE THREAD

**Issue by tashanzhishi** [intel-hadoop/HiBench#391]
when i run the terasort, i find the monitor.html only contain some node, other node monitor infomation dont in the file. my hadoop cluster has 1 master and 6 workers. i only see 3 workers in monitor.html. And i can confirm that the task run all 6 workers.
This is the [terasort.conf](https://drive.google.com/open?id=0B1-hKKHfaUXKTTQ3VXl6Vk9NdU0) and  [monitor.html](https://drive.google.com/open?id=0B1-hKKHfaUXKUC12Nk9CUDR4akE).
if you want get all report file, please click [hadoop.zip](https://drive.google.com/open?id=0B1-hKKHfaUXKVzRuTU5KTVNxcGc).

**Comment by carsonwang** [comment]
For the report log, HiBench detected 6 slaves "cn12 cn14 cn16 cn15 cn13 cn17" correctly but only collected data from three nodes. Three is no data from cn15-cn17. Can you confirm that you were running HiBench on the master node and the master was able to ssh to other slaves without password?

**Comment by tashanzhishi** [comment]
@carsonwang yes, i run hibench in the master node (cn11), and i can ssh cn15~17 without password.
